stable_baselines3/ppo/ppo.py

```diff
@@ -115,7 +118,22 @@ def __init__(
                 spaces.MultiBinary,
             ),
         )
-
+        if self.env is not None:
+            # Checks for scenarios when we are collecting steps from a live environment (self.env not None)
+
+            total_env_steps = len(self.env.envs) * self.n_steps
+            assert total_env_steps > 1, (
+                f"n_steps*n_envs must be greater than 1. " f"Currently n_steps={self.n_steps} and n_envs={len(self.env.envs)}"
+            )
+            untruncated_batches = floor(total_env_steps / batch_size)
+            if total_env_steps % batch_size != 0:
+                warnings.warn(
+                    f"You have specified a desired batch size of {batch_size},"
+                    f" but because your buffer of environment steps will be of size n_steps*n_envs, which"
+                    f" is currently {total_env_steps}, after every {untruncated_batches} untruncated batches,"
```

Comment:
Maybe something simpler: the last mini-batch will be truncated and will be of size ... instead of ....

Reply:
My worry with that framing is that I think the notion of "last mini-batch" is potentially a bit unclear, since we mean it here in the context of "the last mini-batch of every buffer of environment steps," but I don't think that's necessarily the most intuitive parsing (my default, if I saw this error without the context I have, would be to assume it meant the last mini-batch in the whole of training). 

I also worry that explicitly using the framing of "the last mini-batch" to also cover scenarios where we know that this truncation will happen to _every_ mini-batch (because buffer size < batch size) would be extra confusing, since on a sort of language pragmatics level, I expect to only hear something referred to as the "last <thing>" if it's a situation where there's more than one <thing> present.